Mathlib/MeasureTheory/MeasurableSpace/Basic.lean

```diff
@@ -215,7 +215,7 @@ lemma Measurable.sup_of_right {mα mα' : MeasurableSpace α} {_ : MeasurableSpa
 theorem measurable_id'' {m mα : MeasurableSpace α} (hm : m ≤ mα) : @Measurable α α mα m id :=
   measurable_id.mono le_rfl hm
 
--- Porting note (https://github.com/leanprover-community/mathlib4/issues/11215): TODO: add TC `DiscreteMeasurable` + instances
+-- TODO: add typeclass `DiscreteMeasurable` + instances
```

Comment:
I am not sure of what that TODO is referring to, but it might be something like [DiscreteMeasurableSpace](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/MeasurableSpace/Defs.html#DiscreteMeasurableSpace), which we now have?
I'd just remove it.